consensus/state_test.go

```diff
@@ -120,7 +123,8 @@ func TestEnterProposeNoPrivValidator(t *testing.T) {
 	height, round := cs.Height, cs.Round
 
 	// Listen for propose timeout event
-	timeoutCh := subscribeToEvent(cs.evsw, "tester", types.EventStringTimeoutPropose(), 1)
+	timeoutCh := make(chan interface{})
```

Comment:
I didn't like that the only reason was this "one line" benefit... I've renamed it to `subscribe` and reduced the arguments list to just 2 items (`eventBus` and `query`)